### ISSUE TITLE
Relay\Relay::get(): open_basedir restriction in effect. 

### ISSUE THREAD

**Issue by MaximeGratens** [cachewerk/relay#39]
Hello team,

I am using Relay with a Laravel project that has the following versions:

    Laravel 9.21
    Relay 0.4.6
    PHP 8.1

I am using it in production within a Docker environment with the PHP8.1-Relay extension and the serversideup/php:php8.1-fpm-nginx-v1.5.0 image on fly.io.

Sometimes in production, I get the following exception:

```
Relay\Relay::get(): open_basedir restriction in effect. File(/.dockerenv) is not within the allowed path(s): (/var/www/html:/dev/stdout:/tmp)
```

I do not know where it is coming from or how to fix it. Do you have any ideas?

Best regards, and thank you for your work.

**Comment by tillkruss** [comment]
@MaximeGratens: Thanks for reporting this.

@michael-grunder: Can we do a check before calling `php_stat()`?

**Comment by MaximeGratens** [comment]
If you need anything more, mention me :).

**Comment by tillkruss** [comment]
Fixed in v0.6.0
